src/renderer/views/Trending/Trending.js

```diff
@@ -42,13 +52,50 @@ export default Vue.extend({
     }
   },
   mounted: function () {
-    if (this.trendingCache && this.trendingCache.length > 0) {
+    if (this.trendingCache[this.currentTab] && this.trendingCache[this.currentTab].length > 0) {
       this.shownResults = this.trendingCache
     } else {
       this.getTrendingInfo()
     }
   },
   methods: {
+    changeTab: function (tab, event) {
+      if (event instanceof KeyboardEvent) {
+        if (event.key === 'Tab') {
+          return
+        } else if (event.key === 'ArrowLeft' || event.key === 'ArrowRight') {
+          // navigate trending tabs with arrow keys
+          const index = this.tabInfoValues.indexOf(tab)
+          // tabs wrap around from leftmost to rightmost, and vice versa
+          tab = (event.key === 'ArrowLeft')
+            ? this.tabInfoValues[(index > 0 ? index : this.tabInfoValues.length) - 1]
+            : this.tabInfoValues[(index + 1) % this.tabInfoValues.length]
+
+          const tabNode = $(`#${tab}Tab`)
+          event.target.setAttribute('tabindex', '-1')
+          tabNode.attr('tabindex', '0')
+          tabNode.focus()
+        }
+
+        event.preventDefault()
+        if (event.key !== 'Enter' && event.key !== ' ') {
+          return
+        }
+      }
+      const currentTabNode = $('.channelInfoTabs > .tab[aria-selected="true"]')
```

Comment:
I appreciate the mimicry of my tab accessibility changes, but be careful when copying! You actually named the class "trendingInfoTabs".

Reply:
Good catch